classes/patterns/class-action.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Action pattern class.
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Patterns;
+
+/**
+ * Class Action.
+ *
+ * @package P4GBKS\Patterns
+ */
+class Action extends Block_Pattern {
+
+	/**
+	 * Returns the pattern name.
+	 */
+	public static function get_name(): string {
+		return 'p4/action';
+	}
+
+	/**
+	 * Returns the pattern config.
+	 *
+	 * @param array $params Optional array of parameters for the config.
+	 */
+	public static function get_config( $params = [] ): array {
```

Comment:
You can add a `blockTypes` here to add the pattern to Page Creation modal (like [here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/master/classes/patterns/class-blankpage.php#L33))

Reply:
[Updated](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/879/commits/ae8a556d0c2d40e79f5369728ab59a14057b7694)!